doc/specs/thread_tagging.md

```diff
@@ -0,0 +1,215 @@
+# Thread Tagging
+
+## Table of contents
+
+1. Motivation
+2. Business rules
+3. Use cases
+
+## Motivation
+
+The forums are a major component of the site, with a vast wealth of knowledge. 
+Having the ability to tag a thread would make searching much easier and more 
+streamlined, and to help keep certain forums on topic.
+
+## Business Rules
```

Comment:
@ajax-wcd that wasn't specified. I don't believe they're linked in any specific way.